datastore/simple/logger_test.go

```diff
@@ -14,51 +15,28 @@ import (
 var _ = Describe("Proto Logger", func() {
 	var (
 		protoLogger *simple.ProtoLogger
+		logger      *logrus.Logger
```

Comment:
do we need logger here?

Reply:
Revised. This section is a bit complicated due to `BeforeEach` needing to define an outer and local scope variable from a function that returns multiple values.